### PR TITLE
add platform

### PR DESCRIPTION
**Pull request by rokh-conduktor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +47,29 @@ If you want to downgrade confluent platform version, there are two ways:
...
 2. Previous versions have been [built](https://github.com/arm64-compat/confluent-platform) for ARM64 by the community. If you want to use it, just change the image in the corresponding yml. Since it is a not an official image, use it at your own risks.  
 
+## Full stack
+
+To ease you journey with kafka just connect to [localhost:8080](http://localhost:8080/)
+
+login: `login@admin.io`
+password: `admin`
+
+ - Single Zookeeper: `$DOCKER_HOST_IP:2181`
```
Comment (qboileau) [inline README.md:57]: add "- Conduktor-platform : `$DOCKER_HOST_IP:8080`"

File: README.md [hunk 2]
```diff
@@ -46,6 +47,29 @@ If you want to downgrade confluent platform version, there are two ways:
...
+ - Single Kafka: `$DOCKER_HOST_IP:9092`
+ - Kafka Schema Registry: `$DOCKER_HOST_IP:8081`
+ - Kafka Rest Proxy: `$DOCKER_HOST_IP:8082`
+ - Kafka Connect: `$DOCKER_HOST_IP:8083`
+ - KSQL Server: `$DOCKER_HOST_IP:8088`
+ - Zoonavigator Web: `$DOCKER_HOST_IP:8004`
+- (experimental) JMX port at `$DOCKER_HOST_IP:9999`
```
Comment (qboileau) [inline README.md:64]: it's not 9001 ?

File: README.md [hunk 3]
```diff
@@ -46,6 +47,29 @@ If you want to downgrade confluent platform version, there are two ways:
...
+login: `login@admin.io`
+password: `admin`
+
+ - Single Zookeeper: `$DOCKER_HOST_IP:2181`
+ - Single Kafka: `$DOCKER_HOST_IP:9092`
+ - Kafka Schema Registry: `$DOCKER_HOST_IP:8081`
+ - Kafka Rest Proxy: `$DOCKER_HOST_IP:8082`
+ - Kafka Connect: `$DOCKER_HOST_IP:8083`
+ - KSQL Server: `$DOCKER_HOST_IP:8088`
+ - Zoonavigator Web: `$DOCKER_HOST_IP:8004`
```
Comment (qboileau) [inline README.md:63]: it seems that you've drop Zoonavigator from compose file.

File: full-stack.yml [hunk 4]
```diff
@@ -126,11 +126,28 @@ services:
...
+      - 8080:80
+    volumes:
+      - conduktor_data:/var/conduktor
+      - type: bind
+        source: "./platform-config.yaml"
+        target: /opt/conduktor/platform-config.yaml
+        read_only: true
+    depends_on:
+      - kafka1
+      - kafka-schema-registry
```
Comment (qboileau) [inline full-stack.yml:141]: missing `kafka-connect` dependency

File: platform-config.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  name: "${ORGANISATION_NAME}"
+  
+clusters:
+  - id: default
+    name: My Local Kafka Cluster
+    color: "#0013E7"
+    ignoreUntrustedCertificate: false
+    bootstrapServers: "$KAFKA_BOOTSTRAP_SERVER"
+    properties:
+    schemaRegistry:
```
Comment (qboileau) [inline platform-config.yaml:11]: missing kafka connect configuration

File: README.md (hunk 2 above)
Comment (rokh-conduktor) [reply README.md]: +1

File: README.md (hunk 3 above)
Comment (rokh-conduktor) [reply README.md]: +1

File: README.md (hunk 1 above)
Comment (rokh-conduktor) [reply README.md:58]: +1

File: full-stack.yml (hunk 4 above)
Comment (rokh-conduktor) [reply full-stack.yml:141]: +1

**Pull request merged**